ui/components/app/custom-spending-cap/custom-spending-cap.js

```diff
@@ -249,15 +250,16 @@ export default function CustomSpendingCap({
                 'custom-spending-cap__description--with-error-message': error,
               })}
             >
-              <Typography
+              <Text
                 color={TextColor.textDefault}
-                variant={TypographyVariant.H7}
+                variant={TextVariant.bodySm}
+                as="h6"
                 boxProps={{ paddingTop: 2, paddingBottom: 2 }}
```

Comment:
Text doesn't support `boxProps` but you can add the padding directly
```suggestion
                paddingTop={2}
                paddingBottom={2}
```